load/api/viewsets.py

```diff
@@ -21,7 +21,7 @@ def get_queryset(self):
 
     def create(self, request, *args, **kwargs):
         serializer = CreateLoadSerializer(data=request.data)
-        carrier_price_ = request.data["shipper_price"] - request.data["shipper_price"] * 5 / 100
+        carrier_price_ = request.data["shipper_price"] - round(request.data["shipper_price"] * 5 / 100, 2)
```

Comment:
We could create a file utils.py in the load directory for the calculations and call functions from there.